strapi/strapi
# [I18n] RBAC CT & ST refactoring

### What does it do?

This is the first PR of the RBAC view refactoring, it only displays the big permissions table for the create role view. There's no form logic so no checkbox can be checked, the logic will come in another PR.
The layout the we will receive from the backend will change so a plugin will be able to add his own permissions block into a content type section, this PR enables this logic.

> N.B: For the moment I have duplicated the components and refactored a bit, when we start implementing the edition logic most components will move to the `/admin/src` folder instead of the ee one. Also, the settings and plugins tabs are currently empty, it will be added in another PR.

### How to test?

1. Start the development server in entreprise mode
2. Go to the settings > administration panel > role
3. Click on the create button

The view should display 3 content types and under the restaurant section we should be able to see the locales permissions block.
